newscoop/application/configs/parameters/parameters.yml

```diff
@@ -134,3 +134,4 @@ parameters:
         environment: 'prod'
     simple_security:
         users_file: "%application_path%/configs/security/users.json"
+    remember_me.lifetime: 31536000 # 1 year in seconds
```

Comment:
no we cant, the thing is that the lifetime that you set in the backend is not meant to be for remember me functionality but for normal user login, if we will rely on it, the remember me functionality will not work properly because the lifetime will be the same as logging it without "remember me". For the backend the lifetime of remember me is also set in the `security.yml` file. So we either create a new field in system preferences for "remember me" lifetime or we set it in the `custom_parameters.yml` as is in this PR.